cfpb/consumerfinance.gov
# Ask CFPB: restore related resources

Reverses https://github.com/cfpb/consumerfinance.gov/pull/5391, which removed the next steps / related resources section from the Ask answer template. Stakeholders plan to test the use of related resources on top-performing Ask pages.

---


## Additions

- Code to output related resources on Ask answer page template


## Changes

- Update translation files


## How to test this PR

1. Check out branch
2. Create a `Related resource` snippet with both English and Spanish content
3. Add the related resource to both an English and Spanish Ask question
4. Verify that the related resource's English fields are displayed with the heading `Take the next step` on the English page and its Spanish fields are displayed with the translated heading `Dar el siguiente paso` on the Spanish page


## Screenshots
### Related resource

<img width="971" alt="Screen Shot 2022-04-06 at 3 13 56 PM" src="https://user-images.githubusercontent.com/778171/162081221-6f4101ad-9e11-43eb-a77a-891728d3e37c.png">


### English answer page

<img width="888" alt="Screen Shot 2022-04-06 at 3 07 31 PM" src="https://user-images.githubusercontent.com/778171/162080389-199c7da3-5414-428f-9e78-d2a6246bb889.png">

### Spanish answer page

<img width="878" alt="Screen Shot 2022-04-06 at 3 05 34 PM" src="https://user-images.githubusercontent.com/778171/162080369-25abcc90-1aed-4081-80f0-9f7d15257f09.png">



## Notes
I'm not sure why validate-translations is failing -- I *think* I've followed the documented steps for generating translations (latest gettext, `makemessages` & `compilemessages` in virtualenv)
